Subject: Allow build against bitfield 0.17
Author: Daniel Kahn Gillmor <dkg@fifthhorseman.net>

rPGP's use of bitfield is very simple and is not affected by
bitfield's changes.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -58,7 +58,7 @@
 version = ">=0.21.7, < 0.23"
 
 [dependencies.bitfield]
-version = "0.14"
+version = "0.17"
 
 [dependencies.block-padding]
 version = "^0.3.2"
